client/src/app/app.routes.ts

```diff
@@ -5,7 +5,7 @@ import { StatisticsComponent } from "./statistics/statistics.component";
 import { TaskComponent } from "./task/task.component";
 import { TaskEditComponent } from "./task/edit/task-edit.component";
 
-
+/** this is routes block */
```

Comment:
lets **try** to review **it** in code review mode

- [DF-1] make this

- [DF-2] then this